src/addrdb.h

```diff
@@ -43,6 +43,11 @@ class CBanEntry
         nCreateTime = nCreateTimeIn;
     }
 
+    explicit CBanEntry(int64_t n_create_time_in, BanReason ban_reason_in) : CBanEntry(n_create_time_in)
+    {
+        banReason = ban_reason_in;
```

Comment:
nit: could be initializer

Reply:
I don't think this can be initializer as we need to `SetNull()`, the current one does the delegation thing.